### PR TITLE
Polaris December Sync

### PR DESCRIPTION
**Pull request by Cameron653**
Compiles fine and I fixed the few conflicts in the files.

Since it's 200+ files changed, I couldn't look over them all. From what I did see, however, it doesn't look like it'll break the server horribly.

### CODE REVIEW

File: code/game/objects/items/devices/defib.dm [hunk 1]
```diff
@@ -0,0 +1,157 @@
+//The case the paddles are kept in.
+/obj/item/device/defib_kit
+	name = "defibrillator kit"
+	desc = "This KHI-branded defib kit is a semi-automated model. Remove pads, slap on chest, wait."
```
Comment (SpadesNeil) [inline code/game/objects/items/devices/defib.dm]: Oh neat, these are a thing.

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 2]
```diff
@@ -278,3 +278,26 @@
 		beakers += B1
 		beakers += B2
 		icon_state = initial(icon_state) +"_locked"
+
+/obj/item/weapon/grenade/chem_grenade/teargas
+	name = "tear gas grenade"
```
Comment (SpadesNeil) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm]: Oooooo...

File: code/game/objects/items/weapons/implants/implantcase.dm [hunk 3]
```diff
@@ -159,3 +159,23 @@
...
+	desc = "A case containing a GalCom language implant."
+	icon_state = "implantcase-b"
+
+/obj/item/weapon/implantcase/language/New()
+	src.imp = new /obj/item/weapon/implant/language( src )
+	..()
+	return
+
+/obj/item/weapon/implantcase/language/eal
+	name = "glass case - 'EAL'"
```
Comment (SpadesNeil) [inline code/game/objects/items/weapons/implants/implantcase.dm]: These are new! Now we can have in-game reasons for having access to EAL!

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 4]
```diff
@@ -74,7 +74,7 @@
 	new /obj/item/clothing/gloves/yellow(src)
 	new /obj/item/weapon/screwdriver(src)
 	new /obj/item/weapon/wrench(src)
-	new /obj/item/weapon/weldingtool(src)
+	new /obj/item/weapon/weldingtool/experimental(src)
```
Comment (SpadesNeil) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Not sure why this is here. A bit OP, isn't it?

File: code/game/response_team.dm [hunk 5]
```diff
@@ -5,7 +5,7 @@ var/global/send_emergency_team = 0 // Used for automagic response teams
                                    // 'admin_emergency_team' for admin-spawned response teams
 var/ert_base_chance = 10 // Default base chance. Will be incremented by increment ERT chance.
 var/can_call_ert
-var/silent_ert
+var/silent_ert = 0
```
Comment (SpadesNeil) [inline code/game/response_team.dm]: Neat.

File: code/modules/client/preference_setup/loadout/loadout_eyes.dm [hunk 6]
```diff
@@ -38,6 +38,10 @@
 	display_name = "Security HUD, prescription (Security)"
 	path = /obj/item/clothing/glasses/hud/security/prescription
 
+/datum/gear/eyes/security/sunglasshud
+	display_name = "Security HUD, sunglasses (Security)"
```
Comment (SpadesNeil) [inline code/modules/client/preference_setup/loadout/loadout_eyes.dm]: Now the Colony Director can spawn with security glasses if he likes.

File: code/modules/clothing/head/solgov.dm [hunk 7]
```diff
@@ -0,0 +1,241 @@
+//SolGov uniform hats
```
Comment (SpadesNeil) [inline code/modules/clothing/head/solgov.dm]: \o/

File: code/modules/events/apc_damage.dm [hunk 8]
```diff
@@ -14,9 +14,10 @@
 			severity_range = 15
 
 	for(var/obj/machinery/power/apc/apc in range(severity_range,A))
-		if(is_valid_apc(apc))
-			apc.emagged = 1
-			apc.update_icon()
+		if(is_valid_apc(apc))	//This event "****s up" the "id authenticator" on APCs, emagging and unlocking them.
```
Comment (SpadesNeil) [inline code/modules/events/apc_damage.dm]: This should fix the broken APC events.

File: code/modules/mob/living/carbon/human/life.dm [hunk 9]
```diff
@@ -343,14 +344,14 @@
...
-				if(breath.total_moles)
-					if(breath.total_moles < BREATH_MOLES / 5 || breath.total_moles > BREATH_MOLES * 5)
-						rupture_lung()
-				else
-					rupture_lung()
+				rupture_lung()
+
+		if(should_have_organ("brain"))
+			if(prob(5))
+				adjustBrainLoss(0.02 * oxyloss) //2% of your current oxyloss is applied as brain damage, 50 oxyloss is 1 brain damage
```
Comment (SpadesNeil) [inline code/modules/mob/living/carbon/human/life.dm]: Careful everyone. Suffocation has a chance to cause brain damage now.

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 10]
```diff
@@ -167,6 +169,15 @@
...
 	//reagent_tag = IS_SKRELL //VOREStation Removal
+	cold_level_1 = 280 //Default 260 - Lower is better
+	cold_level_2 = 220 //Default 200
+	cold_level_3 = 130 //Default 120
+
+	heat_level_1 = 420 //Default 360 - Higher is better
+	heat_level_2 = 480 //Default 400
+	heat_level_3 = 1100 //Default 1000
+
+	reagent_tag = IS_SKRELL
```
Comment (SpadesNeil) [inline code/modules/mob/living/carbon/human/species/station/station.dm]: Remove this with a Vorestation edit tag.

File: code/modules/mob/living/simple_animal/hostile/bear.dm [hunk 11]
```diff
@@ -129,7 +129,7 @@
 		return
 	custom_emote(1, pick( list("slashes at [target_mob]", "bites [target_mob]") ) )
 
-	var/damage = rand(20,30)
+	var/damage = rand(melee_damage_lower, melee_damage_upper)
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/hostile/bear.dm]: I need to edit the voremob version of bears.

File: code/modules/mob/living/carbon/human/species/station/station.dm (hunk 10 above)
Comment (SpadesNeil) [inline code/modules/mob/living/carbon/human/species/station/station.dm]: @Cameron653

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 12]
```diff
@@ -167,6 +169,15 @@
 	base_color = "#006666"
 
 	//reagent_tag = IS_SKRELL //VOREStation Removal
```
Comment (SpadesNeil) [inline code/modules/mob/living/carbon/human/species/station/station.dm]: @Cameron653

**Comment by Cameron653** [comment]
The "is_skrell" tag has been removed.

**Pull request merged**